linode/manager
# feat: [M3-7487] - Add AGLB Endpoint Health

## Description 📝

- Adds Endpoint Health information to AGLB

## Preview 📷

| Before  | After   |
| ------- | ------- |
| ![Screenshot 2023-12-19 at 3 06 52 PM](https://github.com/linode/manager/assets/115251059/55996eb7-a08e-456f-9323-e4bddeadd427) |  ![Screenshot 2023-12-19 at 3 05 46 PM](https://github.com/linode/manager/assets/115251059/f308f66c-2cad-4462-8ea1-fcd7a0866e0f)   |
|  ![Screenshot 2023-12-19 at 3 07 24 PM](https://github.com/linode/manager/assets/115251059/e102d9fc-e7fe-464f-a54f-b35ac62c0dd8) | ![Screenshot 2023-12-19 at 3 05 39 PM](https://github.com/linode/manager/assets/115251059/4071c490-26f9-4c0b-950f-5406f3a0dbf7) |
| ![Screenshot 2023-12-19 at 3 07 10 PM](https://github.com/linode/manager/assets/115251059/e6600dad-0845-4a74-bbaa-e6635d7abb33)  | ![Screenshot 2023-12-19 at 3 10 39 PM](https://github.com/linode/manager/assets/115251059/27eb5b4c-f982-4d51-bd69-26d28413944a)   |
| ![Screenshot 2023-12-19 at 3 06 59 PM](https://github.com/linode/manager/assets/115251059/3339262e-be8a-4d49-8d97-4259021f0a94) | ![Screenshot 2023-12-19 at 3 05 54 PM](https://github.com/linode/manager/assets/115251059/e3ac2679-d911-41ba-a533-8fa0ee688a8f) |

## How to test 🧪

### Prerequisites
- Use the dev environment 
- Login to the `dev-test-aglb` account

### Verification steps 
- Verify that you see health data in all of the places shown in the `Preview` section

## As an Author I have considered 🤔
